tools/benchmark_tool/README.md

```diff
@@ -128,19 +132,37 @@ Options:
   -shape SHAPE          Optional. Set shape for input. For example,
                         "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]"
                         in case of one input size.
+  -data_shape DATA_SHAPE
+                        Optional. Define shape of data to infer dynamic
+                        model. To process images with original shapes
+                        this parameter can be ommited, but it's required
+                        in other cases to benchmark dynamic model.
+                        For example "[shape1],[shape2], ..." can be
+                        used to set several data shapes in case one input
+                        or to set shape1 to input1, shape2 to input2
+                        and so on in case several inputs. Input mapping
+                        is also supported: "input1[shape1,shape2],input2[shape3,shape4]".
   -layout LAYOUT        Optional. Prompts how network layouts should be
                         treated by application. For example,
                         "input1[NCHW],input2[NC]" or "[NCHW]" in case of one
-                        input size.
+                        input size. Also can be defined partially -
+                        "input1[N...],input2[N...C]"
   -nstreams NUMBER_STREAMS, --number_streams NUMBER_STREAMS
-                       Optional. Number of streams to use for inference on the CPU/GPU/MYX in throughput mode
-                       (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).
-                       Default value is determined automatically for a device.
-                       Please note that although the automatic selection usually provides a reasonable performance,
-                       it still may be non-optimal for some cases, especially for very small networks.
+                        Optional. Number of streams to use for inference on the CPU/GPU/MYX in throughput mode
+                        (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).
+                        Default value is determined automatically for a device.
+                        Please note that although the automatic selection usually provides a reasonable performance,
+                        it still may be non-optimal for some cases, especially for very small networks.
   -nthreads NUMBER_THREADS, --number_threads NUMBER_THREADS
                         Number of threads to use for inference on the CPU
                         (including HETERO  and MULTI cases).
+  --latency_percentile LATENCY_PERCENTILE
+                        Optional. Defines the percentile to be reported in latency metric.
+                        The valid range is [1, 100]. The default value is 50 (median).
+  -enforcebf16 ENFORCEBF16, --enforce_bfloat16 ENFORCEBF16
```

Comment:
`enforcebf16` and `qb` were replaced by `infer_precision` in C++ benchmark_app. Are you planning to align with these changes? https://github.com/openvinotoolkit/openvino/pull/10150/

Reply:
It will be done in a separate PR.